test/integration_test.py

```diff
@@ -590,60 +590,73 @@ def _test_manifest(self, catalog: CatalogName):
             ManifestFormat.verbatim_pfb: self._check_terra_pfb_manifest
         }
         for format in [None, *supported_formats]:
-            # IT catalogs with just one public source are always indexed
-            # completely if that source contains less than the minimum number of
-            # bundles required. So regardless of any randomness employed by this
-            # test, manifests derived from these catalogs will always be based
-            # on the same content hash. Since the resulting reuse of cached
-            # manifests interferes with this test, we need another means of
-            # randomizing the manifest key: a random but all-inclusive filter.
-            tibi_byte = 1024 ** 4
-            filters = {
-                self._file_size_facet(catalog): {
-                    'within': [[0, tibi_byte + self.random.randint(0, tibi_byte)]]
+            for curl in [False, True]:
+                # IT catalogs with just one public source are always indexed
+                # completely if that source contains less than the minimum number of
+                # bundles required. So regardless of any randomness employed by this
+                # test, manifests derived from these catalogs will always be based
+                # on the same content hash. Since the resulting reuse of cached
+                # manifests interferes with this test, we need another means of
+                # randomizing the manifest key: a random but all-inclusive filter.
```

Comment:
Since this was indented, it will need to be re-wrapped. There are more comments like this.